src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -577,6 +579,15 @@ class DocBaseViewer extends BaseViewer {
         // Do not disable create object URL in IE11 or iOS Chrome - pdf.js issues #3977 and #8081 are
         // not applicable to Box's use case and disabling causes performance issues
         PDFJS.disableCreateObjectURL = false;
+
+        // Customize pdf.js loading icon. We modify the prototype of PDFPageView to get around directly modifying
+        // pdf_viewer.js
+        const resetFunc = PDFJS.PDFPageView.prototype.reset;
+        PDFJS.PDFPageView.prototype.reset = function reset(...args) {
+            resetFunc.bind(this)(args);
+            this.loadingIconDiv.classList.add(CLASS_SPINNER);
+            this.loadingIconDiv.innerHTML = '<div></div>';
+        };
```

Comment:
This is so we don't have to go into pdf.js and modify their code after each upgrade.

These are the relevant lines in pdf.js: https://github.com/mozilla/pdf.js/blob/1419b7ffe7fb34aa48002fd388d7a3848c668709/web/pdf_page_view.js#L183